Engine/src/Engine/Scene.cpp

```diff
@@ -93,8 +95,8 @@ namespace Engine
 
 
 		//Setup a callback to update the viewport size when the window is resized
-		glfwSetWindowUserPointer(m_window, reinterpret_cast<void*>(this));
```

Comment:
why are these two lines commented out?

Reply:
To register the callback function with glfw, the user pointer has to be set to an instance of InputSystem. However this would conflict with the window resize callback. Since it was apparently not being used, I commented it out to prevent conflicts. If it is required there is a work around that can be implemented quickly.

Reply:
That makes sense. When we add handling for screen resizing we can apply that through the input system rather than through the scene directly.